breezy/bzr/bzrdir.py

```diff
@@ -1510,7 +1510,7 @@ def _initialize_on_transport_vfs(self, transport):
                 "README",
                 b"This is a Bazaar control directory.\n"
                 b"Do not change any files in this directory.\n"
-                b"See http://bazaar.canonical.com/ for more information about Bazaar.\n",
+                b"See https://launchpad.net/bzr for more information about Bazaar.\n",
```

Comment:
Possibly this one also wants to stay the same - or we're happy changing it as it's not integral to the format but we come back to the question of the tests?

Reply:
I think it would be good writing the new URL when creating new branches, so that users get to the right place.